controls/nist_ocp4.yml

```diff
@@ -15802,6 +15802,14 @@ controls:
 
     https://issues.redhat.com/browse/CMP-544
   rules: []
+- id: SI-6(d)
```

Comment:
This is adding a new control (`SI-6(d)`) and not adding it to any level. So this won't get included in any profile. Let's add it to SI-6, the (d) section is not really a control but merely a bullet point in the control's verbiage.

Reply:
Just fixed it